ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -209,8 +199,8 @@ objects:
                 continue
               fi
               echo "Found router in ${ROUTER_NAMESPACE}"
```

Comment:
This should move out with the `break` too, no?

Reply:
I'm following suit with the other templates in ci-operator/templates/openshift/installer/*  here, do they all need an update?

Reply:
Probably.  If so, I'm fine punting to follow-up work.